lightningd/notification.c

```diff
@@ -1,13 +1,16 @@
 #include <ccan/array_size/array_size.h>
+#include <lightningd/channel.h>
 #include <lightningd/json.h>
 #include <lightningd/notification.h>
+#include <lightningd/peer_htlcs.h>
 
 const char *notification_topics[] = {
 	"connect",
 	"disconnect",
 	"warning",
 	"invoice_payment",
-	"channel_opened"
+	"channel_opened",
+	"forward_event"
```

Comment:
Nit: For future reference, I prefer , on the trailing element, to avoid the extra line change like this.  It's legal since C99 and it's been allowed by compilers for even longer, too.

Reply:
I guess you prefer `autodata` to register notification. :)